### ISSUE TITLE
Can't import self-compiled vapoursynth on W10 with C++2015

### ISSUE THREAD

**Issue by stuxcrystal** [vapoursynth/vapoursynth#259]
What did I do to build:
```
E:\Encoding\Projekte\vapoursynth> msbuild msvc/VapourSynth.sln /p:Configuration=Release
E:\Encoding\Projekte\vapoursynth> call cython_build.bat
```
Builds without problem. (Except for a missing Python 3.6x32 installation)

What happens:

```python 3.6
>>> import os; print(os.get_cwd)
E:\Encoding\Projekte\vapoursynth
>>> import vapoursynth
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: DLL load failed: The specified module could not be found.
```

Software:
Visual C++ 2015 Build Tools
Microsoft Windows 10 Education Version 1607 (Build 14393.576)
Python 3.6.0 x64 for all users.
Python 3.6.0 x32 not installed.

**Comment by myrsloik** [comment]
You don't have vapoursynth.dll in a location where vapoursynth.*.pyd can find it. Put it in the same dir and try again.
I suspect you didn't use the installer...

**Comment by stuxcrystal** [comment]
Now it works.

When I tried to use the debug build before and it didn't work when moving the DLL file.
When I switched to Release I just assumed the problem carried over to the release builds.

Closes #259
